service/mapresolver.go

```diff
@@ -94,23 +97,23 @@ func (mr *mapResolver) Resolve(ctx context.Context) (*config.Map, error) {
 		return nil, fmt.Errorf("cannot close previous watch: %w", err)
 	}
 
-	// Retrieves individual configurations from all locations in the given order, and merge them in retMap.
+	// Retrieves individual configurations from all URIs in the given order, and merge them in retMap.
 	retMap := config.NewMap()
-	for _, location := range mr.locations {
+	for _, uri := range mr.uris {
 		// For backwards compatibility:
 		// - empty url scheme means "file".
 		// - "^[A-z]:" also means "file"
 		scheme := "file"
-		if idx := strings.Index(location, ":"); idx != -1 && !driverLetterRegexp.MatchString(location) {
-			scheme = location[:idx]
+		if idx := strings.Index(uri, ":"); idx != -1 && !driverLetterRegexp.MatchString(uri) {
+			scheme = uri[:idx]
 		} else {
-			location = scheme + ":" + location
+			uri = scheme + ":" + uri
 		}
 		p, ok := mr.mapProviders[scheme]
 		if !ok {
-			return nil, fmt.Errorf("scheme %v is not supported for location %v", scheme, location)
+			return nil, fmt.Errorf("no provider found for the scheme=%v, when resolving the uri=%v", scheme, uri)
```

Comment:
Very minor, but I prefer the older version, with `%q` instead of `%v`. IMO, it's more readable.

Reply:
Done.

Reply:
You want to change to the old message?